ansible_base/authentication/utils/claims.py

```diff
@@ -62,9 +62,13 @@ def create_claims(authenticator: Authenticator, username: str, attrs: dict, grou
         trigger_result = TriggerResult.SKIP
         allowed_keys = TRIGGER_DEFINITION.keys()
         invalid_keys = set(auth_map.triggers.keys()) - set(allowed_keys)
+
+        if auth_map.enabled is False:
+            logger.warning(f"The AuthenticatorMap {auth_map.id} is disabled")
```

Comment:
Maybe info instead? I feel like we're too quick to use warning sometimes - we should save that for things that aren't normal operating conditions. If we're supporting disabling authenticator maps, then that's a normal state for them to be in and warning about it seems excessive.